api/observability/v1/output_types.go

```diff
@@ -210,15 +210,48 @@ type BaseOutputTuningSpec struct {
 
 	// MinRetryDuration is the minimum time to wait between attempts to retry after delivery a failure.
 	//
+	// A duration string is a possibly signed sequence of decimal numbers, each with optional fraction
```

Comment:
Same comment as below

Reply:
> Remove negative example. Do we need to say it's "signed" since only positve numbers make sense. We should additionally add a validation to restrict it to positive numbers

Even if this is straight from the go docs we should still correct the verbiage for our usecase.  IMO we should also restrict it to a precision that is supported by vector.  Do we think that a user can reasonably set it to use microsecond precision?   A negative duration is not valid in our context .  Additionally, I would like to see a validation here, if possilble,  to enforce a postive value

Reply:
Yep, updated.